app/src/main/java/com/celzero/bravedns/adapter/ConnectionTrackerAdapter.kt

```diff
@@ -142,7 +149,10 @@ class ConnectionTrackerAdapter(private val context: Context) :
             // known ports(reserved port and protocol identifiers).
             // https://github.com/celzero/rethink-app/issues/42 - #3 - transport + protocol.
             val resolvedPort = KnownPorts.resolvePort(port)
-            b.connLatencyTxt.text = if (resolvedPort != KnownPorts.PORT_VAL_UNKNOWN) {
+            // case: for UDP/443 label it as HTTP3 instead of HTTPS
+            b.connLatencyTxt.text = if (port == KnownPorts.HTTPS_PORT && proto == Protocol.UDP.protocolType) {
+                HTTP3
```

Comment:
should it be moved to `strings.xml` or Constants?

Reply:
Moved to strings.xml